rdr_service/dao/genomics_dao.py

```diff
@@ -3047,21 +3047,22 @@ def build_latest_decision_query(cls, module: str) -> sqlalchemy.orm.Query:
             )
         )
 
-    def get_latest_state_for_pid(self, pid, module="gem", decision_values_only=True):
+    def get_latest_il_for_pids(self, pid_list, module="gem", decision_values_only=True):
```

Comment:
I am not seeing any reference in the code where this method is being invoked. Are you using this somewhere other than that unit tests?

Reply:
Not anymore, at the moment it is just in the unit tests.